wandb/sdk/data_types/table_decorators.py

```diff
@@ -0,0 +1,88 @@
+"""Decorators for W&B Table operations."""
+
+from functools import wraps
+from typing import Any, Callable, TypeVar, Union
+
+import wandb
+
+T = TypeVar("T")
+
+
+def allow_relogging_after_mutation(method: Callable[..., T]) -> Callable[..., T]:
```

Comment:
Using `...` will make Pylance and MyPy unable to infer type arguments on the annotated methods, weakening type checks throughout the codebase. See the solution in https://github.com/wandb/wandb/pull/9707